.scripts/tester

```diff
@@ -1,12 +1,13 @@
 #!/bin/sh
-
+export SYMFONY_DEPRECATIONS_HELPER=disabled
 FAIL=0
 for D in */; do
   cd $D 
   echo "Operating on $D"
   composer test
   if [ $? -ne 0 ]; then 
     FAIL=1
+    exit 1
```

Comment:
The original point of the `FAIL` variable was to allow all the microservices to run their tests before we reported any failures. If you want to fail on the first error instead then you could remove it entirely.

Reply:
My bad, that's my debug code slipping through, I just wanted a faster test loop and forgot to take it out. I'll remove it.